src/components/AppSettingsDialog/AppSettingsDialog.vue

```diff
@@ -298,11 +298,12 @@ export default {
 		overflow-y: auto;
 		overflow-x: hidden;
 		padding: 24px;
+		width: 100%;
 	}
 }
 
 .navigation-list {
-	height: 100%;
+	height: calc(100% - 2 * 12px);
```

Comment:
Padding compensation? Could `box-sizing: border-box` be useful ?

https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

Reply:
see https://github.com/nextcloud/nextcloud-vue/pull/2197#discussion_r699448026

Reply:
jep, padding compensation. I'll try if your suggestion works.

Reply:
works, fixed